provider/cmd/app/app.go

```diff
@@ -81,13 +79,31 @@ type Options struct {
 
 	Libvirt   LibvirtOptions
 	NicPlugin *networkinterfaceplugin.Options
+	Servers   ServersOptions
 
 	GCVMGracefulShutdownTimeout    time.Duration
 	ResyncIntervalGarbageCollector time.Duration
 
 	VirshExecutable string
 }
 
+type HTTPServerOptions struct {
+	Addr         string
+	ReadTimeout  time.Duration
+	WriteTimeout time.Duration
+	IdleTimeout  time.Duration
+}
+
+type GRPCServerOptions struct {
+	Addr              string
+	ConnectionTimeout time.Duration
+}
+
+type ServersOptions struct {
```

Comment:
it will create conflict with https://github.com/ironcore-dev/libvirt-provider/pull/196